erpnext/crm/doctype/lead/lead.js

```diff
@@ -40,6 +40,32 @@ erpnext.LeadController = frappe.ui.form.Controller.extend({
 
 		if (!this.frm.doc.__islocal) {
 			frappe.contacts.render_address_and_contact(cur_frm);
+
+			let make_quotation = this.frm.dashboard.transactions_area.find('.btn-new[data-doctype="Quotation"]').unbind('click');
```

Comment:
Can we add handler via js hooks on `+` button? Like done for list view events ...